export/export_pipeline/compute_collision_representation.py

```diff
@@ -0,0 +1,427 @@
+            np.random.seed(0)
+            i = 0
+            for p in mesh_parts:
+                p.visual.vertex_colors[:, :3] = (np.random.rand(3) * 255).astype(np.uint8)
+                object_part = trimesh.Scene()
+                object_part.add_geometry(p)
+                filename = osp.join(decompose_dir, f"{i:03d}{params['common']['CONVEX_DECOMPOSITION_EXT']}")
+                object_part.export(filename)
+                object_assembled.add_geometry(p)
+                i+=1
+            if not object_assembled.is_empty:
+                object_assembled.export(osp.join(decompose_dir, 'assembled.obj'))
+                decomposed = True
+        elif acd_args['decompose_method'] == 'vhacd':
+            if not vhacd_args['enable']:
+                return False
+
+            if shutil.which('TestVHACD') is None:
+                print(
+                    "V-HACD was enabled but not found in the system path. Either install it "
+                    "manually or run `bash install_vhacd.sh`. Skipping decomposition"
+                )
+                return False
+
+            # Call V-HACD, suppressing output.
+            fill_mode = {0: 'floor', 1: 'surface', 2: 'raycast'}
+            ret = subprocess.run(
+                [
+                    shutil.which('TestVHACD'),
+                    osp.join('convex_decomposition', mesh_filename),
+                    "-i",
+                    "stl",
+                    "-o",
+                    params['common']['CONVEX_DECOMPOSITION_EXT'][1:],
+                    "-h",
+                    f"{vhacd_args['max_output_convex_hulls']}",
+                    "-r",
+                    f"{vhacd_args['voxel_resolution']}",
+                    "-e",
+                    f"{vhacd_args['volume_error_percent']}",
+                    "-d",
+                    f"{vhacd_args['max_recursion_depth']}",
+                    "-s",
+                    f"{int(not vhacd_args['disable_shrink_wrap'])}",
+                    "-f",
+                    f"{vhacd_args['fill_mode'].name.lower()}",
+                    "-v",
+                    f"{vhacd_args['max_hull_vert_count']}",
+                    "-a",
+                    f"{int(not vhacd_args['disable_asyn'])}",
+                    "-l",
+                    f"{vhacd_args['min_edge_length']}",
+                    "-p",
+                    f"{int(vhacd_args['split_hull'])}",
+                ],
+                check=True,
+                cwd=mesh_dir,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.STDOUT,
+            )
+            output_str = ret.stdout.decode()
+            print(output_str)
+                    
+            if (ret.returncode != 0) or ('VolumeError' in output_str):
+                print(f"V-HACD failed on {mesh_path}")
+            else: 
+                decomposed = True
+            shutil.move(osp.join(mesh_dir, vhacd_args['outputs'][1]), osp.join(decompose_dir, 'assembled.stl'))
```

Comment:
It seems inconsistent to me that some filenames (e.g., `assembled.stl`) are hard-coded in this file but some are specified via the config file.